util/tenant_setup/terraform/lacework/agentless/aws/main.tf

```diff
@@ -0,0 +1,78 @@
+terraform {
+  required_providers {
+    lacework = {
+      source  = "lacework/lacework"
+      version = "1.0.1"
+    }
+  }
+}
+
+variable "lacework_account" {}
+variable "lacework_subaccount" {}
+variable "lacework_apikey" {}
+variable "lacework_apisecret" {}
+
+provider "lacework" {
+  subaccount = var.lacework_subaccount
+  account    = var.lacework_account
+  api_key    = var.lacework_apikey
+  api_secret = var.lacework_apisecret
+}
+
+provider "aws" {
+  alias  = "use1"
+  region = "us-east-1"
+}
+
+provider "aws" {
+  alias  = "use2"
+  region = "us-east-2"
+}
+
+provider "aws" {
+  alias  = "usw1"
+  region = "us-west-1"
+}
+
+provider "aws" {
+  alias  = "usw2"
+  region = "us-west-2"
+}
```

Comment:
Now that you've added these, you'll need to add new region blocks for each. Right now we've only got use1 and usw2 covered. Also for `use1` -> do we want an alias on that? That should be the global provider config i think?

Reply:
Done